### PR TITLE
Update how-to-update-handbook.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Minor editing: headings, sentence structure.

### CODE REVIEW

**Comment by justinegeffen** [comment]
Note: I removed the numbers from the steps so it's not prescriptive/procedural.

File: company/how-to-guides-for-staff/how-to-update-handbook.md [hunk 1]
```diff
@@ -29,19 +29,19 @@ After this, click the **Propose file change** button.
 
 ![](../../.gitbook/assets/how-to-update-handbook-commit-changes.png)
 
-Scroll down to compare changes with the original document, request **Reviewers** on the top right navigation, then select **Create pull request**.
+Scroll down to compare changes with the original document, request **Reviewers** on the top right navigation, select relevant labels for the pull request, then select **Create pull request**.
```
Comment (justinegeffen) [inline company/how-to-guides-for-staff/how-to-update-handbook.md]: I would like to link this out to the code review page for more information about labels and reviewers: https://developers.mattermost.com/contribute/getting-started/code-review/. Thoughts?

Comment (justinegeffen) [reply company/how-to-guides-for-staff/how-to-update-handbook.md]: I noticed that my UI for the PR is different and doesn't have the reviewer and label selection. Is this related to permissions? 
Screen Shot 2020-02-12 at 20 20 52

Comment (jasonblais) [reply company/how-to-guides-for-staff/how-to-update-handbook.md]: @justinegeffen Looks like it's related to permissions. Will work with the team to get that resolved..

Comment (jasonblais) [reply company/how-to-guides-for-staff/how-to-update-handbook.md]: Yes, please link to the labels, thanks!

**Review by jasonblais — APPROVED** [review]
Thanks Justine!

File: company/how-to-guides-for-staff/how-to-update-handbook.md [hunk 2]
```diff
@@ -29,19 +29,19 @@ After this, click the **Propose file change** button.
...
 
-### 4 - Commenting on a Pull Request
+### Commenting on a Pull Request
 
 Once a pull request is submitted, multiple committers may comment on it and provide edits or suggestions which you can commit directly. You can also add line comments. Take a look at [Commenting on pull requests](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request) for more details.
 
-### 5 - Reviewing a Pull Request
+### Reviewing a Pull Request
 
 Once a pull request has been submitted and the correct label assigned, the review process begins. This includes aligning the content with the Style Guide, validating processes, and tagging any other relevant committers.
```
Comment (amyblais) [inline company/how-to-guides-for-staff/how-to-update-handbook.md]:
```suggestion
Once a pull request has been submitted and the correct labels assigned, the review process begins. This includes aligning the content with the Style Guide, validating processes, and tagging any other relevant committers.
```

**Review by amyblais — APPROVED** [review]

**Comment by jasonblais** [comment]
Thanks all!

**Pull request merged**